adding GA events for clicks

jordansread opened the pull request.

Commit:
vizlab.js as resource

Commit:
adding event tracking on outbound clicks

jiwalker-usgs commented on layout/js/animate.js:
```diff
@@ -130,7 +130,7 @@ function openNWIS(id, event){
    event.stopPropagation();
 }else{
-  window.open('http://waterdata.usgs.gov/nwis/uv?site_no='+id,'_blank');
+   window.parent.vizlab.clicklink('http://waterdata.usgs.gov/nwis/uv?site_no='+id,'_blank');
```
Do you need the `.parent`?  I think that was just there because we were embedding svg in object tag.  You should be able to just do `vizlab.clicklink()`

jiwalker-usgs commented on scripts/visualize/visualize.harvey_map.R:
```diff
@@ -211,8 +211,8 @@ visualize.harvey_map <- function(viz = as.viz("harvey-map")){
   wave.d="m48.736 55.595l0.419 0.403c11.752 9.844 24.431 8.886 34.092 2.464 6.088-4.049 33.633-22.367 49.202-32.718v-10.344h-116.03v27.309c7.071-1.224 18.47-0.022 32.316 12.886m43.651 45.425l-13.705-13.142c-1.926-1.753-3.571-3.04-3.927-3.313-11.204-7.867-21.646-5.476-26.149-3.802-1.362 0.544-2.665 1.287-3.586 1.869l-28.602 19.13v34.666h116.03v-24.95c-2.55 1.62-18.27 10.12-40.063-10.46m-44.677-42. ...
-  xml_add_child(g.watermark,'path', d=usgs.d, onclick="window.open('https://www2.usgs.gov/water/','_blank')", 'class'='watermark')
-  xml_add_child(g.watermark,'path', d=wave.d, onclick="window.open('https://www2.usgs.gov/water/','_blank')", 'class'='watermark')
+  xml_add_child(g.watermark,'path', d=usgs.d, onclick="window.parent.vizlab.clicklink('https://www2.usgs.gov/water/')", 'class'='watermark')
```
Here as well I'm not sure if this needs the `window.parent`

jiwalker-usgs commented on viz.yaml:
```diff
@@ -359,12 +363,12 @@ publish:
     scripts: scripts/publish/json.R
     depends: ["footer","footerCSS","header-section", "figure-section", "data-sources", "map-figure-css", 
-      "header-footer-css", "precip-colors", "timesteps_pub", "jquery-js", "animate-js", "timesteps",
+      "header-footer-css", "precip-colors", "timesteps_pub", "jquery-js", "animate-js", "vizlab-js", "timesteps",
```
I was just helping Laura and I'm pretty sure this is going to conflict, but :+1:

jiwalker-usgs reviewed (commented):
Looks good, one question about the scoping of the function is all.

Commit:
adding event for pause/play

Commit:
review feedback

Commit:
merge conflict

Merge branch 'master' of github.com:usgs-vizlab/hurricane-harvey

# Conflicts:
#	viz.yaml
